OutOfSchool/OutOfSchool.WebApi/Services/ApplicationService.cs

```diff
@@ -400,6 +409,18 @@ public async Task<IEnumerable<string>> GetNotificationsRecipientIds(Notification
             return recipientIds.Distinct();
         }
 
+        private async Task<WorkshopStatus> CheckWorkshopStatus(Guid id)
+        {
+            var workshop = await workshopRepository.GetById(id).ConfigureAwait(false);
```

Comment:
The method does not match the name, in my opinion. It simply returns the status, not checks it.